project/ProjectSettings.scala

```diff
@@ -58,6 +58,9 @@ object ProjectSettings {
       .withWarnScalaVersionEviction(false),
   )
 
+  def isCi = sys.env.get("CI").getOrElse("false") == "true"
```

Comment:
I'm curious about this - where does `CI` get set to `true`? This is a new environment variable, is that right?

I don't see any environment variable being set in https://github.com/guardian/frontend/settings/variables/actions :

<img width="1348" alt="image" src="https://github.com/user-attachments/assets/79834397-c1c4-49e1-a67b-67ee7be3b590" />

...I can't see where this `CI` variable would have come from?

Reply:
Oh wow, that's amazing! I did not know about the standard `CI` env variable set by GitHub!

Reply:
I've opened https://github.com/guardian/frontend/pull/27709 to put a tiny bit of logging on this - thank you very much for explaining it to me!